src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/HttpContentJsonExtensions.cs

```diff
@@ -33,7 +33,9 @@ public static Task<T> ReadFromJsonAsync<T>(this HttpContent content, JsonSeriali
 
         private static async Task<object?> ReadFromJsonAsyncCore(HttpContent content, Type type, Encoding? sourceEncoding, JsonSerializerOptions? options, CancellationToken cancellationToken)
         {
+            #pragma warning disable CA2016 // The overload that takes a CancellationToken is not available in netstandard2.0
```

Comment:
Why not

```csharp
#if NETCOREAPP_2_0 // The overload that takes a CancellationToken is not available in netstandard2.0
    Stream contentStream = await content.ReadAsStreamAsync().ConfigureAwait(false);
#else
    Stream contentStream = await content.ReadAsStreamAsync(cancellationToken).ConfigureAwait(false);
#end

Reply:
Good suggestion. Let me just double check if that's the correct macro to use.

Reply:
It should be `NETSTANDARD2_0`.
https://github.com/dotnet/runtime/search?q=NETSTANDARD2_0&unscoped_q=NETSTANDARD2_0

Reply:
Ok thanks, I'm updating it.